runtime/src/lib.rs

```diff
@@ -255,6 +257,8 @@ fn handle_command(drivers: &mut Drivers) -> CaliptraResult<MboxStatusE> {
         }
         #[cfg(feature = "test_only_commands")]
         CommandId::TEST_ONLY_HMAC384_VERIFY => HmacVerifyCmd::execute(drivers, cmd_bytes),
+        #[cfg(feature = "test_only_commands")]
+        CommandId::TEST_ONLY_INC_MIN_SVN => IncMinSvnCmd::execute(drivers),
```

Comment:
This command is a bit odd in that it should never succeed (even as a test-only command). I wonder if it makes more sense as part of one of the tests in `runtime/test-fw` (still as an RT test)."

Could be simple enough to add it as a boot tests (just assert that the RT min SVN slot is locked)